homework-1/task-1/main.js

```diff
@@ -0,0 +1,51 @@
+const inputNumber = document.querySelector('.number');
+const stairsBtn = document.querySelector('.stairs-btn');
+const showStairs = document.querySelector('.show-stairs');
+const lastStairs = document.querySelector('.lastStairs');
+
+let str = '$';
```

Comment:
to common name, change to currencySymbol

Reply:
i think done)))